scripts/pysus_data_upload.py

```diff
@@ -0,0 +1,118 @@
+import os
+import time
+
+import psycopg2
+from pathlib import Path
+
+from dotenv import load_dotenv
+from pandas.io.sql import get_schema
+import pandas as pd
+import psycopg2.extras as extras
+import pyarrow.parquet as pq
+from sqlalchemy import create_engine
+
+from pysus.online_data.SINAN import download
+
+load_dotenv()
+env_path = Path(".") / ".env"
+load_dotenv(dotenv_path=env_path)
+
+
+db_config = {
+    "database": os.getenv('POSTGRES_EPIGRAPH_DB'),
+    "user": os.getenv('POSTGRES_USER'),
+    "password": os.getenv('POSTGRES_PASSWORD'),
+    "host": os.getenv('POSTGRES_HOST'),
+    "port": os.getenv('POSTGRES_PORT'),
+}
+
+PSQL_URI = "postgresql://{}:{}@{}:{}/{}".format(
+    db_config.get('user'),
+    db_config.get('password'),
+    db_config.get('host'),
+    db_config.get('port'),
+    db_config.get('database'),
+
+)
+
+engine = create_engine(PSQL_URI)
+conn = psycopg2.connect(**db_config)
+
+st = time.time()
+
+
+def parquet_to_df(fname: str) -> pd.DataFrame:
+    """
+    Read columns from pyarrowParquet into a Pandas DataFrame.
+
+    Parameters
+    ----------
+        fname: str
+    Returns
+    -------
+        dataframe: pandas
+    """
+    df = (
+        pq.ParquetDataset(f'{fname}/', use_legacy_dataset=False, )
+        .read_pandas() #columns=COL_NAMES
+        .to_pandas()
```

Comment:
Will this read the entire set of Parquet files to memory? or is df a generator? We need to test this with very large parquet files.